kitsune/users/jinja2/users/user_subscriptions.html

```diff
@@ -0,0 +1,25 @@
+{% extends "users/base.html" %}
+{% set title = _("{user} | Subscriptions")|f(user=display_name(user)) %}
+{% set canonical_url = canonicalize(viewname="users.subscriptions", username=user.username) %}
+{% set active = "user-subscriptions" %}
+
+{% block content %}
+  <article id="profile">
+    <h2 class="sumo-page-subheading">
+      {{ _("My Subscriptions") }}
+    </h2>
+    {% for product in products %}
+      <div class="card card--product zoom-on-hover subscriptions">
+        <img class="card--icon" src="{{ product.image_alternate_url }}" alt="{{ pgettext('DB: products.Product.title', product.title) }}">
+        <div class="card--details">
+          <h3 class="card--title">
+            <a class="title" href="{{ url('products.product', product.slug) }}" data-event-category="link click" data-event-action="product" data-event-label="Firefox">
+              {{ _('<strong>{product}</strong>')|fe(product=pgettext('DB: products.Product.title', product.title)) }}
+            </a>
+          </h3>
+          <a href="{{ settings.FXA_SUBSCRIPTIONS }}">Get Support</a>
+        </div>
+      </div>
```

Comment:
Some thoughts on the cards:
- since the whole card isn't clickable, we should remove the `zoom-on-hover` class - as it implies that the card itself is clickable
- IMO the product title link and the get support link are too close together, easy to click the wrong one
- why does the FXA_SUBSCRIPTIONS link say "Get Support", but instead lead users to somewhere where they can manage their subscription?
- could we instead make clicking on the card take a user to step 2 of the AAQ for that product, and have one overall "manage subscriptions" button which takes users to the FXA_SUBSCRIPTIONS link?

Reply:
Regarding the last bullet point let's defer this conversation to the team meeting. For now I will update the string to `Manage Subscriptions` in order to match the functionality that was indicated in the designs. It's an easy change to do later